src/_assets/css/_base.scss

```diff
@@ -72,6 +72,16 @@ dd {
   font: $site-font-icon;
 }
 
+.site-mobile-screenshot {
+  display: block;
+  margin: 0 auto;
+  max-height: none;
+  max-width: 100%;
+  @include media-breakpoint-up(xs) { max-width: breakpoint-min(xxs); }
+
+  &--border { border: 1px solid $site-color-light-grey; }
+}
+
 .site-image-right {
   display: block;
   margin: 0 auto;
```

Comment:
@fertolg - just curious: on the line below the one this comment is attached to, why are you setting the max height to 60vh (and subsequently be forced to set it to none in other subordinate cases)?